libbeat/logp/eventlog_windows.go

```diff
@@ -0,0 +1,91 @@
+package logp
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap/zapcore"
+	"golang.org/x/sys/windows/svc/eventlog"
+)
+
+const (
+	eventID  = 100
```

Comment:
Does `100` have a special meaning in windows event logs?

Reply:
No, it's completely arbitrary. The value needs to be between 1 and 1000. Most of the original log events have three digit IDs so I just chose 100.